src/utils.py

```diff
@@ -32,10 +32,10 @@ def get_last_tag(cwd, _filter: str) -> Union[None, str]:
 
 def compose_build_commit_hash(cwd, tag, files: List, hash_limit=7) -> str:
     commits_hashes = []
-    cmd = ["git", "log", "--format=%h", "--", *files]
+    cmd = ["git", "log", "--format=%H", "--", *files]
```

Comment:
What improvements get this change?

Reply:
-H is printing a full hash, and then the limit is taken from the CLI arguments. with -h it is limited to maximum 9.